src/common/answer.ts

```diff
@@ -5,13 +5,22 @@ export function includes<T extends U, U>(coll: ReadonlyArray<T>, el: U): el is T
 }
 
 export type Answer = Partial<
-    WhereDoYouLive & NationalInsuranceNumberKnown & NationalInsuranceNumber & PensionProviders & TellCivilServicePension
+    NationalInsuranceNumberKnown & NationalInsuranceNumber & PensionProviders & OtherPensionProviders
 >;
 
 export type BooleanQuestion<T extends string> = { [key in T]: 'yes' | 'no' };
 
-export type WhereDoYouLive = { 'where-do-you-live': 'england' | 'scotland' | 'wales' | 'northern-ireland' };
-export type TellCivilServicePension = BooleanQuestion<'tell-civil-service-pension'>;
 export type PensionProviders = { 'pension-providers': 'croydon' | 'sutton' };
 export type NationalInsuranceNumberKnown = BooleanQuestion<'national-insurance-number-known'>;
 export type NationalInsuranceNumber = { 'national-insurance-number': string };
+export type OtherPensionProviders = { 'other-pension-providers': string[] };
+
+export const otherPensionProviderOptions = ['civil', 'armed', 'armed-compensation', 'war', 'none'];
+
+export const OtherPensionProvidersMap: { [index: string]: string } = {
```

Comment:
If the above is an enum, we can make sure we cover all the options here
```suggestion
export const OtherPensionProvidersMap: { [k in OtherPensionProviderOptions]: string } = {
```